analytics.html

```diff
@@ -0,0 +1,206 @@
+<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Crimson+Text" />
+<style>
+    body {
+        background:#232323;
+        color: white;
+        font-family: Arial;
+        margin: 1%;
+        height: 1080px;
+    }
+
+    h2 {
+        font-size: 5em;
+        margin: 0;
+    }
+
+    h3 {
+        font-size: 3.3em;
+        margin: 0;
+    }
+
+    p {
+        font-size: 6.3em;
+        padding-left: 5%;
+        font-weight: 700;
+        margin: 0;
+    }
+
+    .title-bar {
+        display:flex;
+        height: 11%;
+        margin-bottom: 1%;
+    }
+
+    .title-bar div {
+        justify-content: center;
+        font-family: "Crimson Text";
+
+    }
+
+    .box {
+        background:#3a3a3a;
+    }
+
+    #date {
+        flex:1;
+        margin: auto;
+        padding-left: 3%;
+        font-size:2em;
+
+    }
+
+    #title {
+        flex:2;
+        text-align: center;
+        margin: auto;
+        font-size:3.3em;
+    }
+
+    #time {
+        flex:1;
+        margin: auto;
+        padding-right: 3%;
+        text-align: right;
+        font-size: 2em;
+    }
+
+
+    .container {
+        clear:both;
+        display:flex;
+    }
+
+    .left-half{
+        flex:2;
+        order:1;
+        display:flex;
+        flex-direction:column;
+        justify-content: space-between;
+    }
+
+    .left-half div {
+        justify-content: space-around;
+        margin: 1%;
+        padding: 2% 4%;
+    }
+
+    #stats-time-range {
+        font-size: 4em;
+    }
+
+    .right-half{
+        flex:3;
+        order:1;
+        padding:1% 3%;
+        height: 80%;
+        margin-left: 1%;
+    }
+
+    li, .view  {
+        font-family: "crimson-text";
+        font-size: 3em;
+        margin: 1.6em 0;
+    }
+    
+
+    img {
+        font-size: 12px;
+    }
+
+    .articles {
+        display:flex;
+        justify-content: space-between;
+    }
+
+    ol {
+        flex:4;
+        margin: 0;
+    }
+
+    .views {
+        flex:1;
+    }
+
+    .left-text {
+        text-align: left;
+    }
+    .right-text {
+        text-align: right;
+        font-family: arial;
+    }
+
+    .left-text, .right-text {
+        display: inline-block;
+        width: 46.8%;
+
+    }
+
+</style>
+
+<body>
+
+
+    <div class="title-bar box">
+        <div id="date">Tuesday, February 14, 2017</div>
+        <div id="title">Analytics Dashboard</div>
+        <div id="time"><img src="#" alt="Clock icon">3:04pm</div>
+    </div>
+
+    <div class="container">
+        <div class="left-half">
+            <div class="box"> <h3> Statistics from: </h3> <hr/> <p id="stats-time-range">Past Hour</p></div>
+            <div id="current-users" class="box">
+                <h3><img src="#" alt="People icon"> Currently on site </h3>
```

Comment:
Instead of using an image for an icon, look into using font-awesome. I'd ask @psiemens if we have a specific font awesome account we use ubyssey wide, I think we do.

The basic idea behind using font-awesome over loading an image is because an image won't resize/scale nicely where as a unicode icon will cause its treated the same as a letter, afaik. this will also reduce the number of assets we need to load.